chapters/syntax.tex

```diff
@@ -56,13 +56,6 @@ \section{Lexical conventions}\label{lexical-conventions}
 \item
   Modelica uses the same comment syntax as C++ and Java (i.e., \lstinline!//! signals the start of a line comment and \lstinline!/* $\ldots$ */! is a multi-line comment); comments may contain any Unicode character.
   Modelica also has structured comments in the form of annotations and string comments.
-\item
-  Each \lstinline[language=grammar]!description-string! or string in annotations (that is, a \lstinline[language=grammar]!STRING! with production \lstinline[language=grammar]!annotation-clause! in the grammar) may contain any member of the Unicode character set.
-  All other strings have to contain only the subset of Unicode characters identical with the 7-bit US-ASCII character set.
-  \begin{nonnormative}
-  As a consequence, operators like `\lstinline!>!' or `\lstinline!<!', and external functions only operate on ASCII strings and not on Unicode-strings.
-  Within a \lstinline[language=grammar]!description-string! the tags \lstinline!<HTML>! and \lstinline!</HTML>! or \lstinline!<html>! and \lstinline!</html>! define optionally begin and end of content that is HTML encoded.
```

Comment:
I am afraid that this sentence about HTML shouldn't have been removed as part of this PR, as we already have a separate discussion about this in #3051, and we shouldn't mix the two topics.

Reply:
Ok, I restored it then - with grammatical correction and after the list, since it is odd to have a non-normative item in a list and it didn't fit with any other items.
However, I previously explained why I thought it less needed now.

Reply:
Looks good now.  Resolving.